frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -357,7 +358,8 @@ export default ({
         typingUsers: []
       },
       config: {
-        messageMaxChar: CHATROOM_MAX_MESSAGE_LEN
+        messageMaxChar: CHATROOM_MAX_MESSAGE_LEN,
+        debouncedHandleInput: debounce(this.updateTextArea, 250) // NOTE: This is a fix for the issue #2369 and #2577.
```

Comment:
Hey Sebin, I see that this fixes the issue, but I don't understand how or why it works.

Could you add a comment above this line explaining why, for example, this change doesn't break the app? E.g. I thought this `debounce` might mean that keystrokes might get lost but it doesn't. Can you add a comment explaining how this fixes the problem and why no keystrokes are lost?